CONTRIBUTING.md

```diff
@@ -47,6 +47,61 @@ and additionally the following rules:
 * Use `my ($self) = @_;` for parameter parsing in methods when accessing the
   `$self` object. Do not parse any parameter if you do not need any.
 * [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself)
+* Avoid `sleep()`: Do not use `sleep()` or simulate a sleep-like functionality.
+  This is introducing the risk of either unnecessarily wasting time or just
```

Comment:
`s/is introducing/introduces/` or `would introduce`?

Reply:
thx, done